/motor_tester:
  ros__parameters:
    # motor_rid: "t1"
    # goal_type: "pos"
    # goal_upper: 1.7
    # goal_lower: -3.4
    # goal_init: -1.0
    # sensitivity: 1.0 # per second
    # deadzone: 0.2

    motor_rid: "J1"
    goal_type: "pos"
    goal_upper: 3.0
    goal_lower: -3.0
    goal_init: 0.0
    sensitivity: 5.0 # per second
    deadzone: 0.2